_posts/2022-02-21-influencair.md

```diff
@@ -9,4 +9,7 @@ Au travers d'un atelier organisé en collaboration avec l'OpenHub et le Makilab
 
 Ces ateliers et conférences sont ouverts à tous et ne nécessitent pas de connaissances particulières. Plus de détails sont disponibles sur le [site du projet InfluencAir][influencair].
 
+<img src="/img/2022-03-10-influencair-atelier1.jpg" alt="picture1" width="400"/>
```

Comment:
The correct path is now `/img/2022-03-10-influencair/atelier1.jpg` and `/img/2022-03-10-influencair/conference1.jpg`